src/components/Accounts/WalletCard.tsx

```diff
@@ -6,6 +6,7 @@ import { Loadable } from 'recoil'
 import chains from '../../misc/chains'
 import toast from 'react-hot-toast'
```

Comment:
Consider using [Snackbar & Toast](https://mui.com/material-ui/react-snackbar/) so `react-hot-toast` can be removed from dependencies